msom/src/main/webapp/WEB-INF/jsp/modules/list.jsp

```diff
@@ -0,0 +1,35 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>     
+<%@ taglib prefix="sf" uri="http://www.springframework.org/tags/form"%>
+
+<p class="lead">
+    [opis]
```

Comment:
Dobrze byłoby coś od razu tutaj wrzucić, bo jak będziemy pokazywać prowadzącemu i zapomnimy o tym to będzie słabo.